crud/options.go

```diff
@@ -25,86 +19,95 @@ const (
 	batchSizeOptName                     = "batch_size"
 )
 
-type option interface {
-	getInterface() (interface{}, error)
-}
-
 // OptUint is an optional uint.
 type OptUint struct {
-	optional.Uint
+	value uint
+	exist bool
 }
 
-// NewOptUint creates an optional uint from value.
-func NewOptUint(value uint) OptUint {
-	return OptUint{optional.NewUint(value)}
+// MakeOptUint creates an optional uint from value.
+func MakeOptUint(value uint) OptUint {
+	return OptUint{
```

Comment:
Are NewUint more expensive?

Reply:
This is not a very strict rule, but usually:
`NewFoo` return a pointer to a Foo object.
`MakeFoo` return an object of Foo.